code/string_algorithms/src/pangram_checker/README.md

```diff
@@ -0,0 +1,16 @@
+## PANGRAM
+A pangram is a sentence in which every letter of a given alphabet appears at least once. The most famous english pangram is "The quick brown fox jumps over a lazy dog.".
+
+_We only try to implement pangram checker algorithms for sentences in english language._
+
+#### PROCEDURE
```

Comment:
Procedure -> Algorithm

Reply:
Thank you so much for all the suggestions. I really appreciate it.